VideoLocker/src/main/java/org/edx/mobile/view/adapters/BaseListAdapter.java

```diff
@@ -125,6 +125,15 @@ public void replace(T item, int position) {
         notifyDataSetChanged();
     }
 
+    public void moveToTop(int itemPosition) {
+        if (itemPosition < objects.size()) {
+            T item = objects.get(itemPosition);
+            objects.remove(itemPosition);
+            objects.add(0, item);
```

Comment:
In fact instead of the specific `moveToTop()` method, you might define a generic `swapItems()` method that takes two indices explicitly. It's up to you.